features-json/css-image-set.json

```diff
@@ -429,8 +429,8 @@
   "notes":"",
   "notes_by_num":{
     "1":"Has very limited support. Only `url()` is accepted as the image and only `x` is accepted as a resolution.",
-    "2":"Does not support generated images (example: `linear-gradient()`).",
-    "3":"Does not support the `type()` function. "
+    "2":"Does not support generated images in cursor and conten properties (example: `linear-gradient()`) [Bug 1696314](https://bugzilla.mozilla.org/show_bug.cgi?id=1696314).",
```

Comment:
```suggestion
    "2":"Does not support generated images in `content`. Example: `content: image-set(linear-gradient())`. [Bug 1696314](https://bugzilla.mozilla.org/show_bug.cgi?id=1696314).",
```

Reply:
The `cursor` property is still not anything I think we should be tracking. It even seems like the current spec is preferring that browsers don't support it.

The correction, that this issue only applies to `content`, is great. Thank you for pointing that out! I suggest slightly changing the wording as shown above, otherwise this looks good to me :)